app/javascript/helpers/storage_manager/filter-by-capabilities-utils.js

```diff
@@ -0,0 +1,9 @@
+const arrayIncludes = (including, included) => including.length >= included.length
+  && included.every((includedItem) => including.includes(includedItem));
+
+const getCapabilityUuid = (providerCapabilities, capabilityName, capabilityValue) => {
+  const capVals = providerCapabilities[capabilityName];
+  return capVals.find((capVal) => capVal.value === capabilityValue).uuid;
+};
```

Comment:
I would recommend to provide a comment above these function describing what it does.
`/** Function to do such-n-such action to return something */`